dashboard/src/components/ApiDocs/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Auth } from "shared/Auth";
+import SwaggerUI from "swagger-ui-react";
+import "swagger-ui-react/swagger-ui.css";
+
+// Request interface needed for avoiding type error in the requestInterceptor
+interface Request {
+  // tslint:disable-line
+  [k: string]: any;
+}
+
+function authenticate(req: Request): Request {
+  const token = Auth.getAuthToken() || "";
+  req.headers.Authorization = `Bearer ${token}`;
+  req.url = req.url.replace("127.0.0.1:8080", `${window.location.host}`);
+  return req;
+}
```

Comment:
can you try to use the current axios interceptor? (`axiosWithAuth` from `src/shared/AxiosInstance.ts`). This won't work when using OIDC for example.

Reply:
Mmm, I don't get your point here :S 
I just want to exact the bearer token so that it can be used more easily. Otherwise, this token can also be manually entered:
![image](https://user-images.githubusercontent.com/11535726/105846232-2f8ac700-5fdc-11eb-9991-bd0b953fa859.png)

However, the engine for performing the requests is not configurable AFAIK. Look at the props it exposes:

```javascript
    spec?: object | string;
    url?: string;
    onComplete?: (system: System) => void;
    requestInterceptor?: (req: Request) => Request | Promise<Request>;
    responseInterceptor?: (res: Response) => Response | Promise<Response>;
    docExpansion?: 'list' | 'full' | 'none';
    defaultModelExpandDepth?: number;
    plugins?: Plugin[];
    supportedSubmitMethods?: string[];
    deepLinking?: boolean;
```

Reply:
okay, I guess it's okay, the only thing is that I would not add the header if `getAuthToken()` returns nothing (to avoid messing up with the cookies set if oauth2_proxy is enabled).